modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -267,11 +306,28 @@ const Combobox = ({
     focusInput();
   };
 
+  const getGroupIndex = (itemIndex: number | null) => {
+    if (
+      itemIndex != null &&
+      autocompleteItems &&
+      autocompleteItems.length &&
+      'header' in autocompleteItems[0]
```

Comment:
lets change this one too then? `hasOwnProperty`

Reply:
oh shoot I thought I got them all, will do